core/domain/classifier_domain.py

```diff
@@ -365,3 +365,71 @@ def validate(self):
                 raise utils.ValidationError(
                     'Expected answers to be a list, received %s' %
                     grouped_answers['answers'])
+
+
+class ClassifierExplorationMapping(object):
+    """Domain object for a classifier-exploration mapping model.
+
+    A Classifier-Exploration mapping is a one-to-one relation between the
+    attributes in an exploration to the classifier model it needs to utilise.
+    The mapping is from <exp_id, exp_version, state_name> to the classifier_id.
+
+    Attributes:
+        mapping_id: str. The unique id of the classifier-exploration mapping.
```

Comment:
@prasanna08 you would create the composite key on the fly, probably by calling some method in the storage class for that. Then you get by id on the composite key.

@pranavsid98 The domain layer needs to represent the objects we are passing around in the logic layer. There is no requirement of exact matching to the storage model.